broker/src/test/java/io/zeebe/broker/logstreams/AtomixLogDeletionServiceTest.java

```diff
@@ -133,10 +131,8 @@ private void createSnapshot(final long index) {
   private static RaftStorage.Builder builder(
       final RaftStorage.Builder builder, final TemporaryFolder folder) {
     try {
-      final int length = getEntrySize();
       return builder
-          // hardcode max segment size to allow a single entry only
-          .withMaxSegmentSize(JournalSegmentDescriptor.BYTES + 2 * Integer.BYTES + length)
+          .withMaxSegmentSize(JournalSegmentDescriptor.BYTES + 182)
```

Comment:
What is this magic value 182?

Reply:
It's exactly the entry size now, which will change again when we change the serialization. Like I wrote below, `AtomixLogDeletionServiceTest` should be rewritten so it doesn't know about segments, serialization and so on. It has too much implementation knowledge.

Reply:
:+1: